umbraco-cloud/getting-started/migrate-between-regions.md

```diff
@@ -42,81 +42,70 @@ The following steps will guide you through the migration process.
 Make sure that your projects are [prepared for migration](migrate-between-regions.md#prepare-your-projects) before continuing the process.
 {% endhint %}
 
-{% tabs %}
-{% tab title="Umbraco 10+" %}
-1. Clone down the project that you want to migrate - the _EU project_.
-2. Restore content and media through the Umbraco backoffice.
-3. Clone down the new Cloud project created in the US region - the _US project_.
-4. Replace the `src/UmbracoProject/umbraco-cloud.json` file in the _EU project_ with the one from the _US project_.
-
-{% hint style="info" %}
-The `umbraco-cloud.json` file contains details about each environment on the Cloud project.
-
-By replacing the one on the _EU project_ with the one from the _US project_, content, and media transfers will point to the environments on the _US project_ instead of the _EU project_.
-{% endhint %}
-
-5. Commit the change through git, but do not push it yet.
-6. Use the following git commands to connect your local _EU project_ to the live environment on the _US project_:
+### Step 1: Creating and restore database backup
 
-```
-git remote rm origin
-
-git remote add origin https://scm.umbraco.io/useast01/name-of-us-live-site.git
-
-git fetch
-
-git branch --set-upstream-to=origin/master
-```
+{% tab title="Umbraco 10+" %}
```

Comment:
Since the below steps are only for Umbraco 10+, you might want to consider removing the tab markdown.  Also, the steps doesn't appear correctly in  the preview mode but it maybe cz the {% endtabs %} is missing.

Reply:
I just forgot to remove this 🙈

This article is only for v10+ :)